Magento_Checkout/web/js/view/checkout-login-step.js

```diff
@@ -0,0 +1,40 @@
+define([
+  'ko',
+  'uiComponent',
+  'Magento_Checkout/js/model/step-navigator',
+  'Magento_Customer/js/model/customer',
+], function (ko, Component, stepNavigator, customer) {
+  'use strict';
+
+  return Component.extend({
+    defaults: {
+      template: 'Magento_Checkout/check-login',
+    },
+
+    isVisible: ko.observable(true),
+    isLoggedIn: customer.isLoggedIn(),
+    stepCode: 'isLoggedCheck',
+    stepTitle: 'Logging Status',
+
+    initialize: function () {
+      this._super();
+      stepNavigator.registerStep(
+        this.stepCode,
+        null,
+        this.stepTitle,
+        this.isVisible,
+
+        this.navigate.bind(this),
+        1
+      );
+
+      return this;
+    },
+
+    navigate: function () {},
```

Comment:
not sure that this method has to be empty

Reply:
If I delete it the page crash and I get redirected to 404 page